packages/threat-composer/src/components/assumptions/AssumptionCreationCard/index.tsx

```diff
@@ -25,61 +25,66 @@ import MitigationLinkView from '../../mitigations/MitigationLinkView';
 import ThreatLinkView from '../../threats/ThreatLinkView';
 
 export interface AssumptionCreationCardProps {
+  ref?: React.Ref<import('../../generic/GenericEntityCreationCard').GenericEntityCreationCardRef>;
```

Comment:
We do not need dynamic loading here. Move the import to the front `import { GenericEntityCreationCardRef} from '../../generic/GenericEntityCreationCard'`